### DISCUSSION TITLE
High dimensional Fokker-Planck equation

### DISCUSSION THREAD

**Discussion by conan19842002** [mwshinn/PyDDM#37] [Ask for help] [answered]
How do I define a high dimensional Fokker-Planck (model) with this package. The Model class does not seem to support this

**Comment by mwshinn** [comment, marked as answer]
This is not currently possible - right now, PyDDM only supports one-dimensional diffusion.

**Comment by conan19842002** [comment]
Hi Max,

Thank you for letting me know. Could you provide a guide on how to extend
the package to solve for  multi-dimensional systems?

Many thanks,
An


---



An Nguyen


https://www.linkedin.com/in/annguyen1984

**Reply by mwshinn** [reply]
Unfortunately, no, I can't provide you with a guide.  This is a very major extension that will take on the order of months for someone who is already familiar with the codebase.  Figuring out how to best go about it is a major feat in itself, let alone implementing it.
